main/menus.js

```diff
@@ -175,9 +185,20 @@ const applicationMenuTemplate = [
   }
 ];
 
-const cogMenu = Menu.buildFromTemplate(cogMenuTemplate);
+let cogMenu = Menu.buildFromTemplate(getCogMenuTemplate());
 const cogExportsItem = cogMenu.getMenuItemById('exports');
 
+const refreshRecordPluginItems = services => {
+  pluginsItems = services.map(service => ({
+    label: service.title,
+    type: 'checkbox',
+    checked: service.isEnabled,
+    click: service.toggleEnbaled
+  }));
+
+  cogMenu = Menu.buildFromTemplate(getCogMenuTemplate());
+};
```

Comment:
This is a workaround, since I can't grab the item itself and change submenu. I have to keep recreating the menu itself